### DISCUSSION TITLE
Negative `margin-bottom` not working anymore.

### DISCUSSION THREAD

**Discussion by pedrolimasi** [polybar/polybar#2641] [Q&A] [answered]
I have in my config
```ini
[global/wm]
margin-bottom = -12
```
but after updating polybar to `3.6.1` that is not being imposed. Positive values work fine, but negative values are the same as inputting zero.

Anyone else noticed it? Is there another way to get a similar behavior?

**Comment by patrick96** [comment, marked as answer]
Looks like an oversight on our part. Will convert to an issue and fix this for version 3.6.2

**Reply by patrick96** [reply]
Fixed in #2644
